.github/workflows/slow-checks.yml

```diff
@@ -0,0 +1,43 @@
+name: Slow-Checks
+
+on:
+  workflow_call:
+    secrets:
+      ALTERNATIVE_GITHUB_TOKEN:
+        required: false
+
+jobs:
+
+  Tests:
+    name: Integration-Tests (Python-${{ matrix.python-version }}, Exasol-${{ matrix.exasol-version}})
+    runs-on: ubuntu-latest
+    # Even though the environment "manual-approval" will be created automatically,
+    # it still needs to be configured to require interactive review.
+    # See project settings on GitHub (Settings / Environments / manual-approval).
+    environment: manual-approval
+    env:
+      GITHUB_TOKEN: ${{ secrets.ALTERNATIVE_GITHUB_TOKEN ||  secrets.GITHUB_TOKEN }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ "3.9", "3.10", "3.11", "3.12" ]
+        exasol-version: [ "7.1.9" ]
```

Comment:
This version seems to be chosen randomly.
I would suggest to use latest 7.1.30 and 8.32.0

Reply:
As discussed, this will be addressed in a separate PR where matrix generation will be added in a more generic way.